File: src/FacturXDotNet.WebEditor/src/app/core/facturx-models/cii/buyer-order-referenced-document.ts
/**
 * **PURCHASE ORDER REFERENCE**
 *
 * @ID BT-13-00
 * @CiiXmlPath /rsm:CrossIndustryInvoice/rsm:SupplyChainTradeTransaction/ram:ApplicableHeaderTradeAgreement/ram:BuyerOrderReferencedDocument
 * @Profile MINIMUM
 */
export interface BuyerOrderReferencedDocument {
  /**
   * **Purchase order reference** - An identifier of a referenced purchase order, issued by the Buyer.
   *
   * @ID BT-13
   * @CiiXmlPath /rsm:CrossIndustryInvoice/rsm:SupplyChainTradeTransaction/ram:ApplicableHeaderTradeAgreement/ram:BuyerOrderReferencedDocument/ram:IssuerAssignedID
   * @Profile MINIMUM
   * @ChorusPro
   * For the public sector, this is the "Engagement Juridique" (Legal Commitment). It is mandatory for some buyers. You should refer to the ChorusPro Directory
   * to identify these public entity buyers that make it mandatory.
   */
  issuerAssignedId?: string;
}

File: src/FacturXDotNet/Models/CII/BuyerOrderReferencedDocument.cs
namespace FacturXDotNet.Models.CII;

/// <summary>
///     <b>PURCHASE ORDER REFERENCE</b>
/// </summary>
/// <ID>BT-13-00</ID>
/// <CiiXPath>/rsm:CrossIndustryInvoice/rsm:SupplyChainTradeTransaction/ram:ApplicableHeaderTradeAgreement/ram:BuyerOrderReferencedDocument</CiiXPath>
/// <Profile>MINIMUM</Profile>
public class BuyerOrderReferencedDocument
{
    /// <summary>
    ///     <b>Purchase order reference</b> - An identifier of a referenced purchase order, issued by the Buyer.
    /// </summary>
    /// <ID>BT-13</ID>
    /// <CiiXPath>/rsm:CrossIndustryInvoice/rsm:SupplyChainTradeTransaction/ram:ApplicableHeaderTradeAgreement/ram:BuyerOrderReferencedDocument/ram:IssuerAssignedID</CiiXPath>
    /// <Profile>MINIMUM</Profile>
    /// <ChorusPro>
    ///     For the public sector, this is the "Engagement Juridique" (Legal Commitment). It is mandatory for some buyers. You should refer to the ChorusPro Directory
    ///     to identify these public entity buyers that make it mandatory.
    /// </ChorusPro>
    public string? IssuerAssignedId { get; set; }
}
